---
title: "Change log"
output: html_document
---


# List of R tools for microbiome data analysis  

## Change log 13 October, 2021     

* Other Tools   
    * Added gggenes, gggenomes      
    * Added ggplot2 gallery    
* Microbiome Tools    
    * rANOMALY   
    * SuRF   
    * timeOmics   
    * CMMB   
    * MiATDS   
    * RobZS   

## Change log 11 September, 2021     

* Added Contributors section      
* Added resources new 8 to 10  
* Added 'Additional' resources suggested by Svetlana UP to Controls section  
* Added OTU/ASV debate Controls section   

## Change log 08 August, 2021     

* Tools    
    * diffTop and microeco added   
* GitHub actions testing for website rendering  

## Change log 04 August, 2021     

* Added Tutorials and Educational Resources    
    
## Change log 22 May, 2021     

* Revamped website    
* Tools    
    * metaSPARSim added   
* References for Controls in Microbiome Research  
    
## Change log 07 May, 2021     

* Tools    
    * ANCOM-BC added next to ANCOM    
    * ALDEx2 [Analysis Of Differential Abundance Taking Sample Variation Into Account](https://bioconductor.org/packages/release/bioc/html/ALDEx2.html)    
    * CoDaSeq [Functions for compositional analysis of high throughput sequencing](https://github.com/ggloor/CoDaSeq)     
    * DivNet [An R package to estimate diversity when taxa in the community cooccur via a ecological network](https://github.com/adw96/DivNet)  
    

## Change log 20 March, 2021  

* Tools    
    * MicrobiomeExplor [An R package for the analysis and visualization of microbial communities](https://doi.org/10.1093/bioinformatics/btaa838)   
    * MicroBVS [Dirichlet-tree multinomial regression models with Bayesian variable selection - an R package](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03640-0)    
    * MicroNiche [An R package for assessing microbial niche breadth and overlap from amplicon sequencing data](https://academic.oup.com/femsec/article-abstract/96/8/fiaa131/5863182?redirectedFrom=fulltext)  
    * NMIT [Microbial Interdependence Association Test--a Non-parametric Microbial Interdependence Test](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5696116/#!po=75.0000)    
    * NetCoMi [Network construction and comparison for microbiome data in R](https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbaa290/6017455)
    * NBZIMM [Negative binomial and zero-inflated mixed models, with application to microbiome/metagenomics data analysis](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03803-z)    
    * selbal [Balances: a New Perspective for Microbiome Analysis](https://msystems.asm.org/content/3/4/e00053-18)  
    * tidyMicro [A pipeline for microbiome data analysis and visualization using the tidyverse in R](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-03967-2)   
    
    
* Microbiome data sets  
    * HMP16SData [Efficient Access to the Human Microbiome Project through Bioconductor](https://bioconductor.org/packages/release/data/experiment/html/HMP16SData.html)   
    * HMP2Data [16s rRNA sequencing data from the Human Microbiome Project 2](https://bioconductor.org/packages/release/data/experiment/html/HMP2Data.html)     *microbiomeDataSets [Experiment Hub based microbiome datasets](https://bioconductor.org/packages/devel/data/experiment/html/microbiomeDataSets.html)   
    * curatedMetagenomicData [Accessible, curated metagenomic data through ExperimentHub](https://bioconductor.org/packages/release/data/experiment/html/curatedMetagenomicData.html)    
    
    
* Other tools
    * [Antoni Susin, Yiwen Wang, Kim-Anh LÃª Cao, M.Luz Calle.: Variable selection in microbiome compositional data analysis: tutorial](https://malucalle.github.io/Microbiome-Variable-Selection/)  
    
* Reformatted the code for website  
* Added Changes log  
* Added Acknowledgments   
    












Shetty, Sudarshan A., and Leo Lahti. [Microbiome data science. Journal of biosciences 44, no. 5 (2019): 115](https://link.springer.com/article/10.1007%2Fs12038-019-9930-2).    
[*Pre-print*](https://openresearchlabs.github.io/publications/papers/2019-Shetty-MDS.pdf)   
Zendo: [![DOI](https://zenodo.org/badge/99698135.svg)](https://zenodo.org/badge/latestdoi/99698135)    
