---
title: ""
pagetitle: "Tools for microbiome anlaysis" 
output: html_document
---



## A list of R environment based tools for microbiome data exploration, statistical analysis and visualization  
  
Microbiome data are challenging to analyse. Development of tools and resources for microbiome data science are ever increasing. This website is a resource for researchers to know about the available tools and resources. This is in no way a complete list and we try our best to search, identify and list the tools and resources. 

Please fill the survey to let us know your thoughts about this resource. Google Forms link https://forms.gle/NZzmgtMe9EfuQXvn6   

-------------------------------------------------------    

> A   

Adaptive gPCA [A method for structured dimensionality reduction](https://arxiv.org/abs/1702.00501)     
Ampvis2 [Tools for visualising amplicon sequencing data](https://madsalbertsen.github.io/ampvis2/)   
AMR [An R Package for Working with Antimicrobial Resistance Data](https://msberends.github.io/AMR/)   
ANCOM/ANCOM-BC [R package for Analysis of Composition of Microbiomes (ANCOM-BC)](https://github.com/FrederickHuangLin/ANCOMBC)   
animalcules [R shiny app for interactive microbiome analysis](https://compbiomed.github.io/animalcules-docs/)   
ALDEx2 [Analysis Of Differential Abundance Taking Sample Variation Into Account](https://bioconductor.org/packages/release/bioc/html/ALDEx2.html)  
adaANCOM [Transformation and differential abundance analysis of microbiome data incorporating phylogeny](https://doi.org/10.1093/bioinformatics/btab543)   

> B    

BDMMA [Batch Effects Correction for Microbiome Data With Dirichlet-multinomial Regression](https://academic.oup.com/bioinformatics/article-abstract/35/5/807/5078473)   
BEEM [BEEM: Estimating Lotka-Volterra models from time-course microbiome sequencing data](https://github.com/lch14forever/BEEM)   
biome-shiny [GUI for microbiome visualization, based on the shiny package "microbiome"](https://github.com/Biodata-PT/Biome-Shiny)   
bootLong [The Block Bootstrap Method for Longitudinal Microbiome Data](https://pratheepaj.github.io/bootLong/)   
breakaway [Species Richness Estimation and Modeling](https://github.com/search?q=breakaway)   
benchdamic [benchmarking of differential abundance methods for microbiome data](https://bioconductor.org/packages/benchdamic/)   
BiomeHorizon [Visualizing Microbiome Time Series Data in R](https://doi.org/10.1128/msystems.01380-21)   

> C   

CCREPE [Compositionality Corrected by PErmutation and REnormalization](http://bioconductor.org/packages/release/bioc/html/ccrepe.html)   
corncob [Modeling microbial abundances and dysbiosis with beta-binomial regression](https://arxiv.org/abs/1902.02776)  
curatedMetagenomicData [Accessible, curated metagenomic data through ExperimentHub](https://waldronlab.io/curatedMetagenomicData/)    
CoDaSeq [Functions for compositional analysis of high throughput sequencing](https://github.com/ggloor/CoDaSeq)     
C3NA [correlation and consensus-based cross-taxonomy network analysis for compositional microbial data](https://doi.org/10.1186/s12859-022-05027-9)   
CoDaCoRe [Learning Sparse Log-Ratios for High-Throughput Sequencing Data](https://doi.org/10.1093/bioinformatics/btab645)  
cmmb [A Compositional Mediation Model for Binary Outcome: Application to Microbiome Studies](https://doi.org/10.1093/bioinformatics/btab605)  

> D     

dacomp [Testing for Differential Abundance in Compositional Counts Data, with Application to Microbiome Studies](https://github.com/barakbri/dacomp)   
DADA2 [Divisive Amplicon Denoising Algorithm](https://www.nature.com/nmeth/journal/v13/n7/full/nmeth.3869.html)    
DECIPHER [Using DECIPHER v2.0 to Analyze Big Biological Sequence Data in R](http://www2.decipher.codes/index.html)   
DECIPHER/IIDTAXA [IDTAXA: a novel approach for accurate taxonomic classification of microbiome sequences](doi:10.1186/s40168-018-0521-5)   
decontam [Simple Statistical Identification and Removal of Contaminant Sequences in Marker-Gene and Metagenomics Data](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-018-0605-2)   
DESeq2 [Differential expression analysis for sequence count data](https://www.bioconductor.org/packages/devel/bioc/vignettes/phyloseq/inst/doc/phyloseq-mixture-models.html)  
diffTop [Differential topic analysis](https://pratheepaj.github.io/diffTop/)  
DivNet [An R package to estimate diversity when taxa in the community cooccur via a ecological network](https://github.com/adw96/DivNet)  
DMBC [A Dirichlet-Multinomial Bayes Classifier for Disease Diagnosis With Microbial Compositions](https://msphere.asm.org/content/2/6/e00536-17)    
dysbiosisR [Calculate Microbiome Dysbiosis](https://microsud.github.io/dysbiosisR/)  

> E   

edgeR [empirical analysis of DGE in R](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2796818/)    
endoR [Interpreting tree ensemble machine learning models with endoR](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010714)   
EasyMicroPlot [An Efficient and Convenient R Package in Microbiome Downstream Analysis and Visualization for Clinical Study](https://doi.org/10.3389/fgene.2021.803627)   
EzMAP [Easy Microbiome Analysis Platform](https://doi.org/10.1186/s12859-021-04106-7)   

> F

FuncDiv [Contributional diversity of joint taxa-function data](https://github.com/gavinmdouglas/FuncDiv)

> G     

GLMMMiRKAT [A distance-based kernel association test based on the generalized linear mixed model](https://github.com/hk1785/GLMM-MiRKAT)   
GMEmbeddings [An R Package to Apply Embedding Techniques to Microbiome Data](https://doi.org/10.3389/fbinf.2022.828703)   
ggpicrust2 [integrates pathway annotations and 10 DA methods](https://github.com/cafferychen777/ggpicrust2)  

> I   

igraph [Network Analysis and Visualization in R](http://igraph.org/r/)    
iNMF [A zero-inflated non-negative matrix factorization for the deconvolution of mixed signals of biological data](https://doi.org/10.1515/ijb-2020-0039)  

> L    

labdsv [Ordination and Multivariate Analysis for Ecology](https://cran.r-project.org/web/packages/labdsv/labdsv.pdf)  
LinDA [A log-normal regression and differential abundance too for counts data that corrects for library size to infer the nature of zeros] (https://github.com/zhouhj1994/LinDA)
LOCOM [A logistic regression model for testing differential abundance in compositional microbiome data with false discovery rate control](https://doi.org/10.1073/pnas.2122788119)   
LDM [A rarefaction-without-resampling extension of PERMANOVA for testing presence-absence associations in the microbiome](https://doi.org/10.1093/bioinformatics/btac399)

> M   

MaAsLin2 [MaAsLin2: Microbiome Multivariate Association with Linear Models](https://github.com/biobakery/Maaslin2)   
mare [Microbiota Analysis in R Easily](https://github.com/katrikorpela/mare)   
massMap [A Two-Stage Microbial Association Mapping Framework With Advanced FDR Control](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-018-0517-1)  
mbtools [Collection of R tools to analyze microbiome data](https://gibbons-lab.github.io/mbtools)   
MDiNE [MDiNE: a model to estimate differential co-occurrence networks in microbiome studies](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz824/5614428)   
MDPbiome [MDPbiome: microbiome engineering through prescriptive perturbations](https://academic.oup.com/bioinformatics/article/34/17/i838/5093255)    
MedTest [A Distance-Based Approach for Testing the Mediation Effect of the Human Microbiome](https://academic.oup.com/bioinformatics/article/34/11/1875/4810437)    
MegaR [An Interactive R Package for Metagenomic Sample Classification and Disease Prediction using Microbiota and Machine Learning](https://github.com/BioHPC/MegaR) 
MelonnPan [Model-based Genomically Informed High-dimensional Predictor of Microbial Community Metabolic Profiles](https://github.com/biobakery/melonnpan)  
metabaRpipe [A DADA2 python container to quickly process 16S amplicons](https://github.com/fconstancias/metabaRpipe)
Metacoder [An R package for visualization and manipulation of community taxonomic diversity data](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005404)   
metagenomeSeq [Differential abundance analysis for microbial marker-gene surveys](http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2658.html)   
MetaLonDA [METAgenomic LONgitudinal Differential Abundance method](https://github.com/aametwally/MetaLonDA)   
metamicrobiomeR [Analysis of Microbiome Relative Abundance Data using Zero Inflated Beta GAMLSS and Meta-Analysis Across Studies using Random Effects Model](https://github.com/cran/metamicrobiomeR)    
metaSPARSim [An R tool for 16S rRNA-gene sequencing count data simulation](https://gitlab.com/sysbiobig/metasparsim)   
MiATDS [A powerful adaptive microbiome-based association test for microbial association signals with diverse sparsity levels](https://doi.org/10.1016/j.jgg.2021.08.002)
microbiome R package  [Tools for microbiome analysis in R](https://github.com/microbiome/microbiome)
MicrobiomeDDA [An Omnibus Test for Differential Distribution Analysis of Microbiome Sequencing Data](https://academic.oup.com/bioinformatics/article/34/4/643/4470360)
MicrobiomeExplor [An R package for the analysis and visualization of microbial communities](https://doi.org/10.1093/bioinformatics/btaa838)   
MicrobiomeHD [A standardized database of human gut microbiome studies in health and disease *Case-Control*](http://www.biorxiv.org/content/early/2017/05/08/134031) 
microbiomeMarker [R package for microbiome biomarker discovery](https://github.com/yiluheihei/microbiomeMarker)   
MicrobiomeProfiler [An R/shiny package for microbiome functional enrichment analysis](https://github.com/YuLab-SMU/MicrobiomeProfiler)    
MicrobiomeR [MicrobiomeR: An R Package for Simplified and Standardized Microbiome Analysis Workflows](https://microbiomer.vallenderlab.org/)    
MicrobiomeStat [MicrobiomeStat: Statistical Methods for Microbiome Compositional Data](https://cran.r-project.org/web/packages/MicrobiomeStat/index.html)     
microbiomeutilities [Extending and supporting package based on microbiome and phyloseq R package](https://microsud.github.io/microbiomeutilities/)   
microbiotaPair [An R Package for Simplified Microbiome Analysis Workflows on Intervention Study](https://github.com/rusher321/microbiotaPair)    
MicrobiotaProcess [MicrobiotaProcess: an R package for analysis, visualization and biomarker discovery of microbiome](https://github.com/YuLab-SMU/MicrobiotaProcess)   
MicroBVS [Dirichlet-tree multinomial regression models with Bayesian variable selection - an R package](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03640-0)    
microDecon [An R package for removing contamination from metabarcoding (e.g., microbiome) datasets post-sequencing](https://github.com/donaldtmcknight/microDecon)  
MicroNiche [An R package for assessing microbial niche breadth and overlap from amplicon sequencing data](https://academic.oup.com/femsec/article-abstract/96/8/fiaa131/5863182?redirectedFrom=fulltext)   
microRMS [Reduced metagenome sequencing for strain-resolution taxonomic profiles](https://doi.org/10.1186/s40168-021-01019-8)   
microshades [An R Package for Improving Color Accessibility and Organization of Microbiome Data](https://doi.org/10.1128/mra.00795-22)  
microViz [An R package for for analysis and visualization of microbiome sequencing data](https://david-barnett.github.io/microViz/)
MiDAT [An adaptive direction-assisted test for microbiome compositional data](https://doi.org/10.1093/bioinformatics/btac361)   
miLineage [A General Framework for Association Analysis of Microbial Communities on a Taxonomic Tree](https://medschool.vanderbilt.edu/tang-lab/software/miLineage)
MIMOSA2 [A metabolic network-based tool for inferring mechanism-supported relationships in microbiome-metabolome data](https://doi.org/10.1093/bioinformatics/btac003)
mina [Microbial community dIversity and Network Analysis with mina](https://github.com/Guan06/mina)   
MINT [Multivariate INTegrative method](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1553-8)   
MiRKAT-MC [A Distance-Based Microbiome Kernel Association Test With Multi-Categorical Outcomes](https://doi.org/10.3389/fgene.2022.841764)   
mixDIABLO [Data Integration Analysis for Biomarker discovery using Latent variable approaches for ‘Omics studies](http://mixomics.org/mixdiablo/)   
mixMC [Multivariate Statistical Framework to Gain Insight into Microbial Communities](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0160169)   
MMinte [Methodology for the large-scale assessment of microbial metabolic interactions (MMinte) from 16S rDNA data](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1230-3)   
MTA [Microbial trend analysis (MTA) for common dynamic trend, group comparison and classification in longitudinal microbiome study](https://github.com/chanw0/MTA)   
MZINBVA [Variational approximation for multilevel zero-inflated negative-binomial models for association analysis in microbiome surveys](https://doi.org/10.1093/bib/bbab443)   

> N    
 
NMIT [Microbial Interdependence Association Test--a Non-parametric Microbial Interdependence Test](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5696116/#!po=75.0000)    
NetCoMi [Network construction and comparison for microbiome data in R](https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbaa290/6017455)   
NBZIMM [Negative binomial and zero-inflated mixed models, with application to microbiome/metagenomics data analysis](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03803-z)    

> O    

OMiSA [Optimal Microbiome-based Survival Analysis (OMiSA)](https://github.com/hk1785/OMiSA)   
OmicsLonDA [Robust identification of temporal biomarkers in longitudinal omics studies](https://doi.org/10.1093/bioinformatics/btac403)   

> P   

pathostat [Statistical Microbiome Analysis on metagenomics results from sequencing data samples](https://bioconductor.org/packages/release/bioc/html/PathoStat.html) - [support deprecated](https://github.com/mani2012/PathoStat/issues/48)
phylofactor [Phylogenetic factorization of compositional data](https://peerj.com/articles/2969/)   
phylogeo [Geographic analysis and visualization of microbiome data](https://www.ncbi.nlm.nih.gov/pubmed/25913208)   
phyloMDA [an R package for phylogeny-aware microbiome data analysis](https://doi.org/10.1186/s12859-022-04744-5)  
phyloseq [Import, share, and analyze microbiome census data using R](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0061217)   
Pldist [Pldist: Ecological Dissimilarities for Paired and Longitudinal Microbiome Association Analysis](https://academic.oup.com/bioinformatics/article-abstract/35/19/3567/5341424?redirectedFrom=fulltext)   
powmic [Power assessment in microbiome case-control studies](https://github.com/lichen-lab/powmic)   
POST [Phylogeny-guided microbiome OTU-specific association test (POST)](https://doi.org/10.1186/s40168-022-01266-3)
PQLSeq [An efficient tool designed for differential analysis of large-scale RNA sequencing (RNAseq) data and Bisulfite sequencing (BSseq) data in the presence of individual relatedness and population structure](https://cran.r-project.org/web/packages/PQLseq/index.html)
prokounter [Differential richness inference for 16S rRNA marker gene surveys](https://github.com/mskb01/prokounter)   

> Q   

qgraph [Graph Plotting Methods, Psychometric Data Visualization and Graphical Model Estimation](https://cran.r-project.org/web/packages/qgraph/qgraph.pdf)   
qiime2R [Importing QIIME2 artifacts and associated data into R sessions](https://github.com/jbisanz/qiime2R)   
qiimer [R tools compliment qiime](https://github.com/kylebittinger/qiimer)    

> R    

RAM [R for Amplicon-Sequencing-Based Microbial-Ecology](https://cran.r-project.org/web/packages/RAM/RAM.pdf)   
Rbec [a tool for analysis of amplicon sequencing data from synthetic microbial communities](https://www.nature.com/articles/s43705-021-00077-1)    
RCM [A unified framework for unconstrained and constrained ordination of microbiome read count data](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0205474)   
RevEcoR [Reverse Ecology analysis in R](https://github.com/yiluheihei/RevEcoR)   
Rhea [A pipeline with modular R scripts](https://peerj.com/articles/2836/)    
RobZS [Sparse least trimmed squares regression with compositional covariates for high dimensional data](https://doi.org/10.1093/bioinformatics/btab572)   

> S   

selbal [Balances: a New Perspective for Microbiome Analysis](https://msystems.asm.org/content/3/4/e00053-18)  
ShinyPhyloseq [Web-tool with user interface for Phyloseq](http://joey711.github.io/shiny-phyloseq/)     
SIAMCAT [Statistical Inference of Associations between Microbial Communities And host phenoTypes](https://bioconductor.org/packages/release/bioc/html/SIAMCAT.html)   
SigTree [Identify and Visualize Significantly Responsive Branches in a Phylogenetic Tree](http://www.sciencedirect.com/science/article/pii/S2001037017300132)    
SparseMCMM [Estimating and testing the microbial causal mediation effect with the high-dimensional and compositional microbiome data (SparseMCMM)](https://sites.google.com/site/huilinli09/software)   
SPIEC-EASI [Sparse and Compositionally Robust Inference of Microbial Ecological Networks](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004226)   
SplinectomeR [SplinectomeR Enables Group Comparisons in Longitudinal Microbiome Studies](https://www.frontiersin.org/articles/10.3389/fmicb.2018.00785/full)   
StructFDR [False Discovery Rate Control Incorporating Phylogenetic Tree Increases Detection Power in Microbiome-Wide Multiple Testing](https://academic.oup.com/bioinformatics/article/33/18/2873/3824757)   
structSSI [Simultaneous and Selective Inference for Grouped or Hierarchically Structured Data](https://www.jstatsoft.org/article/view/v059i13)  
specificity [an R package for analysis of feature specificity to environmental and higher dimensional variables, applied to microbiome species data](https://doi.org/10.1186/s40793-022-00426-0)   

> T   

Tax4Fun [Predicting functional profiles from metagenomic 16S rRNA gene data](https://www.ncbi.nlm.nih.gov/pubmed/25957349)    
Tax4Fun2 [Prediction of habitat-specific functional profiles and functional redundancy based on 16S rRNA gene sequences](https://doi.org/10.1186/s40793-020-00358-7)    
taxize [Taxonomic Information from Around the Web](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3901538/)    
TreeSummarizedExperiment [An extension class of SummarizedExperiment to allow hierarchical structure on the row or column dimension](https://github.com/fionarhuang/TreeSummarizedExperiment)   
themetagenomics [Exploring Thematic Structure and Predicted Functionality of 16S rRNA Amplicon Data](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0219235)    
tidyMicro [A pipeline for microbiome data analysis and visualization using the tidyverse in R](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-03967-2)   
Tweedieverse [Differential analysis of omics data based on the Tweedie distribution](https://github.com/himelmallick/Tweedieverse)    
timeOmics [An R package for longitudinal multi-omics data integration](https://doi.org/10.1093/bioinformatics/btab664)  

> V   

vegan  [R package for community ecologists](https://github.com/vegandevs/vegan)   

> Y   

yingtools2 [Tools and functions for working with clinical and microbiome data](https://github.com/ying14/yingtools2)   

> Z    

zeroSum [Reference Point Insensitive Molecular Data Analysis](https://academic.oup.com/bioinformatics/article/33/2/219/2928229)   
ZIBBSeqDiscovery [A Zero-inflated Beta-binomial Model for Microbiome Data Analysis](https://onlinelibrary.wiley.com/doi/abs/10.1002/sta4.185)   

-------------------------------------------------------    

### Microbiome data sets  

1. HMP16SData [Efficient Access to the Human Microbiome Project through Bioconductor](https://bioconductor.org/packages/release/data/experiment/html/HMP16SData.html)   
2. HMP2Data [16s rRNA sequencing data from the Human Microbiome Project 2](https://bioconductor.org/packages/release/data/experiment/html/HMP2Data.html)  
3. microbiomeDataSets [Experiment Hub based microbiome datasets](https://bioconductor.org/packages/devel/data/experiment/html/microbiomeDataSets.html)   
4. curatedMetagenomicData [Accessible, curated metagenomic data through ExperimentHub](https://bioconductor.org/packages/release/data/experiment/html/curatedMetagenomicData.html)   

-------------------------------------------------------    

Help us spread information about tools for microbiome data analytics by sharing it with others too!   
You can cite this resource as:   
Shetty, Sudarshan A., and Leo Lahti. [Microbiome data science. Journal of biosciences 44, no. 5 (2019): 115](https://link.springer.com/article/10.1007%2Fs12038-019-9930-2).    
[*Pre-print*](https://openresearchlabs.github.io/publications/papers/2019-Shetty-MDS.pdf)   

-------------------------------------------------------    

### Do you want to join us and contribute to the open source microbiome community?   
We are working on an open source project developing R/Bioc methods, benchmarking data, and educational material for microbiome research based on the `SummarizedExperiment` class and its derivatives.  
More information please check our website [R/Bioc microbiome ecosystem with SummarizedExperiment](https://microbiome.github.io/)   

-------------------------------------------------------    

### Acknowledgments        

* The Academy of Finland (Grants 295741; 307127)     
* The SIAM Gravitation Grant 024.002.002   
* UNLOCK project of the Netherlands Organization for Scientific Research (NWO)   
* University of Turku, Department of Mathematics and Statistics   
* Molecular Ecology group, Laboratory of Microbiology, Wageningen University, The Netherlands   
* Medische Microbiologie en Infectiepreventie, University Medical Center Groningen, The Netherlands   

-------------------------------------------------------     

### Updates   

* Updates that are added [See Changes log](CHANGES.html)   
  

-------------------------------------------------------   

Zendo:  
![DOI](https://zenodo.org/badge/99698135.svg)[](https://zenodo.org/badge/latestdoi/99698135)    


Site format from [Dr. Emily C. Zabor](https://github.com/zabore/tutorials)
