<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Change log</title>

<script src="site_libs/header-attrs-2.20/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Tools for Microbiome Analysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="https://microbiome.github.io/">
    <span class="fa fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="TutorialsResources.html">Tutorials Resources</a>
</li>
<li>
  <a href="OtherTools.html">Other Tools</a>
</li>
<li>
  <a href="ControlsMicrobiome.html">About Controls</a>
</li>
<li>
  <a href="Contributors.html">Contributors</a>
</li>
<li>
  <a href="CHANGES.html">Updates</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://scholar.google.nl/citations?hl=en&amp;user=Vahc6LUAAAAJ&amp;view_op=list_works&amp;sortby=pubdate">
    <span class="ai ai-google-scholar ai-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://github.com/microsud">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/gutmicrobe">
    <span class="fab fa-twitter fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="www.linkedin.com/in/sudarshan-shetty-57345822">
    <span class="fab fa-linkedin fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Change log</h1>

</div>


<div id="list-of-r-tools-for-microbiome-data-analysis"
class="section level1">
<h1>List of R tools for microbiome data analysis</h1>
<div id="change-log-11-april-2023" class="section level2">
<h2>Change log 11 April, 2023</h2>
<ul>
<li>Microbiome Tools
<ul>
<li>ggpicrust2<br />
</li>
<li>mEQO</li>
</ul></li>
</ul>
</div>
<div id="change-log-28-december-2022" class="section level2">
<h2>Change log 28 December, 2022</h2>
<ul>
<li>Microbiome Tools
<ul>
<li>adaANCOM<br />
</li>
<li>benchdamic<br />
</li>
<li>C3NA<br />
</li>
<li>microshades<br />
</li>
<li>GMEmbeddings<br />
</li>
<li>prokounter<br />
</li>
<li>LOCOM<br />
</li>
<li>endoR<br />
</li>
<li>specificity</li>
<li>OmicsLonDA</li>
<li>LDM<br />
</li>
<li>POST<br />
</li>
<li>phyloMDA<br />
</li>
<li>MiDAT<br />
</li>
<li>BiomeHorizon<br />
</li>
<li>dysbiosisR<br />
</li>
<li>MiRKAT-MC<br />
</li>
<li>EasyMicroPlot<br />
</li>
<li>MIMOSA2<br />
</li>
<li>MZINBVA<br />
</li>
<li>timeOmics<br />
</li>
<li>CoDaCoRe<br />
</li>
<li>cmmb<br />
</li>
<li>MiATDS<br />
</li>
<li>RobZS<br />
</li>
<li>Tax4Fun2<br />
</li>
<li>EzMAP<br />
</li>
<li>iNMF<br />
</li>
<li>microRMS</li>
</ul></li>
<li>Other Tools
<ul>
<li>DataExplorer</li>
</ul></li>
</ul>
</div>
<div id="change-log-24-december-2021" class="section level2">
<h2>Change log 24 December, 2021</h2>
<ul>
<li>Tutorials and Educational Resources
<ul>
<li>Hannah Freund. Amplicon Analysis Workflow<br />
</li>
<li>Tully, B. J. et al. BVCN<br />
</li>
</ul></li>
<li>Microbiome Tools
<ul>
<li>MicrobiomeStat<br />
</li>
<li>Rbec<br />
</li>
<li>AMR<br />
</li>
<li>MicrobiomeProfiler<br />
</li>
<li>Tweedieverse<br />
</li>
<li>MINA<br />
</li>
</ul></li>
<li>Other Tools
<ul>
<li>TreeHeatmap<br />
</li>
<li>Fastverse</li>
</ul></li>
</ul>
</div>
<div id="change-log-13-october-2021" class="section level2">
<h2>Change log 13 October, 2021</h2>
<ul>
<li>Other Tools
<ul>
<li>Added gggenes, gggenomes<br />
</li>
<li>Added ggplot2 gallery<br />
</li>
</ul></li>
<li>Microbiome Tools
<ul>
<li>rANOMALY<br />
</li>
<li>SuRF<br />
</li>
<li>timeOmics<br />
</li>
<li>CMMB<br />
</li>
<li>MiATDS<br />
</li>
<li>RobZS</li>
</ul></li>
</ul>
</div>
<div id="change-log-11-september-2021" class="section level2">
<h2>Change log 11 September, 2021</h2>
<ul>
<li>Added Contributors section<br />
</li>
<li>Added resources new 8 to 10<br />
</li>
<li>Added ‘Additional’ resources suggested by Svetlana UP to Controls
section<br />
</li>
<li>Added OTU/ASV debate Controls section</li>
</ul>
</div>
<div id="change-log-08-august-2021" class="section level2">
<h2>Change log 08 August, 2021</h2>
<ul>
<li>Tools
<ul>
<li>diffTop and microeco added<br />
</li>
</ul></li>
<li>GitHub actions testing for website rendering</li>
</ul>
</div>
<div id="change-log-04-august-2021" class="section level2">
<h2>Change log 04 August, 2021</h2>
<ul>
<li>Added Tutorials and Educational Resources</li>
</ul>
</div>
<div id="change-log-22-may-2021" class="section level2">
<h2>Change log 22 May, 2021</h2>
<ul>
<li>Revamped website<br />
</li>
<li>Tools
<ul>
<li>metaSPARSim added<br />
</li>
</ul></li>
<li>References for Controls in Microbiome Research</li>
</ul>
</div>
<div id="change-log-07-may-2021" class="section level2">
<h2>Change log 07 May, 2021</h2>
<ul>
<li>Tools
<ul>
<li>ANCOM-BC added next to ANCOM<br />
</li>
<li>ALDEx2 <a
href="https://bioconductor.org/packages/release/bioc/html/ALDEx2.html">Analysis
Of Differential Abundance Taking Sample Variation Into Account</a><br />
</li>
<li>CoDaSeq <a href="https://github.com/ggloor/CoDaSeq">Functions for
compositional analysis of high throughput sequencing</a><br />
</li>
<li>DivNet <a href="https://github.com/adw96/DivNet">An R package to
estimate diversity when taxa in the community cooccur via a ecological
network</a></li>
</ul></li>
</ul>
</div>
<div id="change-log-20-march-2021" class="section level2">
<h2>Change log 20 March, 2021</h2>
<ul>
<li>Tools
<ul>
<li>MicrobiomeExplor <a
href="https://doi.org/10.1093/bioinformatics/btaa838">An R package for
the analysis and visualization of microbial communities</a><br />
</li>
<li>MicroBVS <a
href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03640-0">Dirichlet-tree
multinomial regression models with Bayesian variable selection - an R
package</a><br />
</li>
<li>MicroNiche <a
href="https://academic.oup.com/femsec/article-abstract/96/8/fiaa131/5863182?redirectedFrom=fulltext">An
R package for assessing microbial niche breadth and overlap from
amplicon sequencing data</a><br />
</li>
<li>NMIT <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5696116/#!po=75.0000">Microbial
Interdependence Association Test–a Non-parametric Microbial
Interdependence Test</a><br />
</li>
<li>NetCoMi <a
href="https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbaa290/6017455">Network
construction and comparison for microbiome data in R</a></li>
<li>NBZIMM <a
href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03803-z">Negative
binomial and zero-inflated mixed models, with application to
microbiome/metagenomics data analysis</a><br />
</li>
<li>selbal <a
href="https://msystems.asm.org/content/3/4/e00053-18">Balances: a New
Perspective for Microbiome Analysis</a><br />
</li>
<li>tidyMicro <a
href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-03967-2">A
pipeline for microbiome data analysis and visualization using the
tidyverse in R</a></li>
</ul></li>
<li>Microbiome data sets
<ul>
<li>HMP16SData <a
href="https://bioconductor.org/packages/release/data/experiment/html/HMP16SData.html">Efficient
Access to the Human Microbiome Project through Bioconductor</a><br />
</li>
<li>HMP2Data <a
href="https://bioconductor.org/packages/release/data/experiment/html/HMP2Data.html">16s
rRNA sequencing data from the Human Microbiome Project 2</a>
*microbiomeDataSets <a
href="https://bioconductor.org/packages/devel/data/experiment/html/microbiomeDataSets.html">Experiment
Hub based microbiome datasets</a><br />
</li>
<li>curatedMetagenomicData <a
href="https://bioconductor.org/packages/release/data/experiment/html/curatedMetagenomicData.html">Accessible,
curated metagenomic data through ExperimentHub</a></li>
</ul></li>
<li>Other tools
<ul>
<li><a
href="https://malucalle.github.io/Microbiome-Variable-Selection/">Antoni
Susin, Yiwen Wang, Kim-Anh Lê Cao, M.Luz Calle.: Variable selection in
microbiome compositional data analysis: tutorial</a></li>
</ul></li>
<li>Reformatted the code for website<br />
</li>
<li>Added Changes log<br />
</li>
<li>Added Acknowledgments</li>
</ul>
<p>Shetty, Sudarshan A., and Leo Lahti. <a
href="https://link.springer.com/article/10.1007%2Fs12038-019-9930-2">Microbiome
data science. Journal of biosciences 44, no. 5 (2019): 115</a>.<br />
<a
href="https://openresearchlabs.github.io/publications/papers/2019-Shetty-MDS.pdf"><em>Pre-print</em></a><br />
Zendo: <a href="https://zenodo.org/badge/latestdoi/99698135"><img
src="https://zenodo.org/badge/99698135.svg" alt="DOI" /></a></p>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
