<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Controls for Microbiome Research</title>

<script src="site_libs/header-attrs-2.20/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Tools for Microbiome Analysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="https://microbiome.github.io/">
    <span class="fa fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="TutorialsResources.html">Tutorials Resources</a>
</li>
<li>
  <a href="OtherTools.html">Other Tools</a>
</li>
<li>
  <a href="ControlsMicrobiome.html">About Controls</a>
</li>
<li>
  <a href="Contributors.html">Contributors</a>
</li>
<li>
  <a href="CHANGES.html">Updates</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://scholar.google.nl/citations?hl=en&amp;user=Vahc6LUAAAAJ&amp;view_op=list_works&amp;sortby=pubdate">
    <span class="ai ai-google-scholar ai-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://github.com/microsud">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/gutmicrobe">
    <span class="fab fa-twitter fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="www.linkedin.com/in/sudarshan-shetty-57345822">
    <span class="fab fa-linkedin fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Controls for Microbiome Research</h1>

</div>


<p>Useful resources for microbiome sequencing especially for 16S rRNA
gene-based studies.<br />
<strong>NOTE</strong>: The resources mentioned here are in no way
comprehensive and if you know any articles/resources please highlight it
on the <a
href="https://github.com/microsud/Tools-Microbiome-Analysis/issues">GitHub
repo</a>.</p>
<blockquote>
<p>Contamination</p>
</blockquote>
<ol style="list-style-type: decimal">
<li><a
href="https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-014-0087-z">Salter
SJ, Cox MJ, Turek EM, Calus ST, Cookson WO, Moffatt MF, Turner P,
Parkhill J, Loman NJ, Walker AW. Reagent and laboratory contamination
can critically impact sequence-based microbiome analyses. BMC biology.
2014 Dec 1;12(1):87</a>.<br />
</li>
<li><a href="https://msystems.asm.org/content/4/4/e00290-19">Karstens L,
Asquith M, Davin S, Fair D, Gregory WT, Wolfe AJ, Braun J, McWeeney S.
Controlling for contaminants in low-biomass 16S rRNA gene sequencing
experiments. MSystems. 2019 Aug 27;4(4)</a>.<br />
</li>
<li><a href="doi:10.1186/s13099-016-0103-7">Glassing A, Dowd SSE,
Galandiuk S, Davis B, Chiodini RJ. 2016. Inherent bacterial DNA
contamination of extraction and sequencing reagents may affect
interpretation of microbiota in low bacterial biomass samples. Gut
Pathog 8:24</a>.<br />
</li>
<li><a href="doi:10.1371/journal.pone.0110808">Lusk RW. 2014. Diverse
and widespread contamination evident in the unmapped depths of high
throughput sequencing data. PLoS One 9:e110808</a>.<br />
</li>
<li><a href="doi:10.1186/s40168-015-0083-8">Jervis-Bardy J, Leong LEX,
Marri S, Smith RJ, Choo JM, Smith-Vaughan HC, Nosworthy E, Morris PS,
O’Leary S, Rogers GB, Marsh RL. 2015. Deriving accurate microbiota
profiles from human samples with low bacterial content through
post-sequencing processing of Illumina MiSeq data. Microbiome
3:19</a>.<br />
</li>
<li><a href="doi:10.1186/s13099-016-0103-7">Glassing A, Dowd SSE,
Galandiuk S, Davis B, Chiodini RJ. 2016. Inherent bacterial DNA
contamination of extraction and sequencing reagents may affect
interpretation of microbiota in low bacterial biomass samples. Gut
Pathog 8:24</a>.<br />
</li>
<li><a href="doi:10.1186/s13059-014-0564-2">Weiss S, Amir A, Hyde ER,
Metcalf JL, Song SJ, Knight R. 2014. Tracking down the sources of
experimental contamination in microbiome studies. Genome Biol
15:564</a>.<br />
</li>
<li><a href="doi:10.1128/mSystems.00163-18">Caruso V, Song X, Asquith M,
Karstens L. 2019. Performance of microbiome sequence inference methods
in environments with varying biomass. mSystems 4:e00163-18</a>.<br />
</li>
<li><a href="https://doi.org/10.1186/s12866-020-01795-7">Claassen-Weitz,
S., Gardner-Lubbe, S., Mwaikono, K.S. et al. Optimizing 16S rRNA gene
profile analysis from low biomass nasopharyngeal and induced sputum
specimens. BMC Microbiol 20, 113 (2020)</a>.<br />
</li>
<li><a href="https://doi.org/10.1038/s41598-019-55568-y">Douglas, C.A.,
Ivey, K.L., Papanicolas, L.E. et al. DNA extraction approaches
substantially influence the assessment of the human breast milk
microbiome. Sci Rep 10, 123 (2020)</a>.<br />
</li>
<li><a href="https://doi.org/10.1186/s40168-020-00998-4">Moossavi, S.,
Fehr, K., Khafipour, E. et al. Repeatability and reproducibility
assessment in a large-scale population-based microbiota study: case
study on human milk microbiota. Microbiome 9, 41 (2021)</a>.</li>
</ol>
<blockquote>
<p>Positive controls</p>
</blockquote>
<ol style="list-style-type: decimal">
<li><a href="doi:10.1128/mSystems.00062-16">Bokulich NA, Rideout JR,
Mercurio WG, Shiffer A, Wolfe B, Maurice CF, Dutton RJ, Turnbaugh PJ,
Knight R, Caporaso JG. 2016. mockrobiota: a public resource for
microbiome bioinformatics benchmarking. mSystems
1(5):e00062-16</a>.<br />
</li>
<li><a href="https://doi.org/10.1038/s41596-020-0362-0">Cichocki, N.,
Hübschmann, T., Schattenberg, F. et al. Bacterial mock communities as
standards for reproducible cytometric microbiome analysis. Nat Protoc
15, 2788–2812 (2020)</a>.<br />
</li>
<li><a href="https://doi.org/10.1038/sdata.2016.81">Singer, E.,
Andreopoulos, B., Bowers, R. et al. Next generation sequencing data of a
defined microbial mock community. Sci Data 3, 160081 (2016)</a>.<br />
</li>
<li><a href="https://doi.org/10.1128/mSystems.00023-18">Yeh Y-C, Needham
DM, Sieradzki ET, Fuhrman JA. 2018. Taxon disappearance from microbiome
analysis reinforces the value of mock communities as a standard in every
sequencing run. mSystems 3:e00023-18</a>.<br />
</li>
<li><a href="https://doi.org/10.1186/s12866-016-0738-z">Fouhy, F.,
Clooney, A.G., Stanton, C. et al. 16S rRNA gene sequencing of mock
microbial populations- impact of DNA extraction method, primer choice
and sequencing platform. BMC Microbiol 16, 123 (2016)</a>.</li>
</ol>
<blockquote>
<p>Food for thought</p>
</blockquote>
<ol style="list-style-type: decimal">
<li><a href="https://doi.org/10.1128/AEM.02627-17">Pollock J,
Glendinning L, Wisedchanwet T, Watson M. 2018. The madness of
microbiome: attempting to find consensus “best practice” for 16S
microbiome studies. Appl Environ Microbiol 84:e02627-17</a>.<br />
</li>
<li><a href="https://doi.org/10.1186/s40168-021-01059-0">Nearing, J.T.,
Comeau, A.M. &amp; Langille, M.G.I. Identifying biases and their
potential solutions in human microbiome studies. Microbiome 9, 113
(2021)</a>.<br />
</li>
<li><a href="https://doi.org/10.1093/femsec/fiz045">Bastian V H Hornung,
Romy D Zwittink, Ed J Kuijper, Issues and current standards of controls
in microbiome research, FEMS Microbiology Ecology, Volume 95, Issue 5,
May 2019</a><br />
</li>
<li><a
href="http://fiererlab.org/2018/08/15/garbage-in-garbage-out-wrestling-with-contamination-in-microbial-sequencing-projects/">Noah
Fierer, Jessica Henley, and Matt Gebert 2018. Garbage in, garbage out:
Wrestling with contamination in microbial sequencing projects</a></li>
</ol>
<blockquote>
<p>Additional</p>
</blockquote>
<ol style="list-style-type: decimal">
<li><a
href="https://onlinelibrary.wiley.com/doi/full/10.1111/1755-0998.13011">Weyrich
et al., 2019 Laboratory contamination over time during low-biomass
sample analysis</a><br />
</li>
<li><a href="https://elifesciences.org/articles/46923">McLaren et al.,
2019 Consistent and correctable bias in metagenomic sequencing
experiments</a><br />
</li>
<li><a href="https://www.nature.com/articles/s41585-018-0104-z">Karstens
et al., 2018 Community profiling of the urinary microbiota:
considerations for low-biomass samples</a><br />
</li>
<li><a
href="https://sfamjournals.onlinelibrary.wiley.com/doi/full/10.1111/lam.13091">Stinson
et al., 2018 Identification and removal of contaminating microbial DNA
from PCR reagents: impact on low-biomass microbiome analyses</a><br />
</li>
<li><a
href="https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-018-0605-2">Davis
et al., 2018 Simple statistical identification and removal of
contaminant sequences in marker-gene and metagenomics data</a><br />
</li>
<li><a href="https://www.nature.com/articles/s41564-018-0202-y">de
Goffau et al., 2018 Recognizing the reagent microbiome</a><br />
</li>
<li><a
href="https://www.cell.com/action/showPdf?pii=S0966-842X%2818%2930253-1">Eisenhofer
et al., 2018 Contamination in Low Microbial Biomass Microbiome Studies:
Issues and Recommendations</a><br />
</li>
<li><a
href="https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-017-0267-5">Kim
et al., 2017 Optimizing methods and dodging pitfalls in microbiome
research</a></li>
</ol>
<blockquote>
<p>OTU/ASV debate</p>
</blockquote>
<ol style="list-style-type: decimal">
<li><a
href="http://fiererlab.org/2020/02/17/whats-in-a-number-estimating-microbial-richness-using-dada2/">Corinne
Walsh &amp; Noah Fierer 2020. What’s in a number? Estimating microbial
richness using DADA2</a><br />
</li>
<li><a href="https://forum.mothur.org/t/otus-vs-asvs/3506">Theory behind
mothur OTU/ASV</a><br />
</li>
<li><a
href="Lumping%20versus%20splitting%20–%20is%20it%20time%20for%20microbial%20ecologists%20to%20abandon%20OTUs?">Noah
Fierer, Tess Brewer, &amp; Mallory Choudoir</a></li>
</ol>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
